ProjectEureka/project-eureka-flutter
# PE-40 | Implement The Rating Page

**What I did:**

- Designed a rating system where the user can rate their experience with another user.
- User can touch and drag along a star bar, rating between 0 and 5.
- When user is done they can click the **DONE!** button to submit their rating.
- By default, if the user clicks on the DONE! button without giving a rating, it will auto fill zero for the rating.
- The rating is displayed while the user is giving a rating and will display again in console when rating is submitted.
- When rating is submitted user is routed back to homepage.
- The design look and feel is similar to what we agreed on Figma.

**How to Test:**

1. Pull from branch
2. Run flutter pub get
3. Test to make sure that ratings bar can slide from left to right.
4. Make sure the number rating is indicating the proper rating as you slide on the screen.
5. Once you submit make sure the rating you gave is displayed on your console.
6. Verify that  you're sent back to the homepage after, you click the **Done!** button.
